### PR TITLE
Updates to process_mon.sh and ubuntu_updates_check.sh

### PR DESCRIPTION
**Pull request by tcaddy**
## Updates to process_mon.sh
Add an additional metric `process_age` to the `process_mon.sh` script.  This will allow you to raise an alarm if a process has been running for too long.

## Updates to ubuntu_update_check.sh
* `/usr/lib/update-notifier/apt-check` was not returning a count of upgradeable packages on various machines even though you could see packages available for update via `apt-get upgrade`
 * Replace with `apt list --upgradeable` and parse out the count of upgradeable security and non-security package updates available

### CODE REVIEW

File: process_mon.sh [hunk 1]
```diff
@@ -64,4 +68,12 @@ if [ -z "$1" ]; then
 fi
 
 process_check=`ps ax | grep -v grep | grep -v process_mon | grep -c "$1"`
+if (( $process_check > 0 )); then
+  process_age=`ps -o etimes= -p "$process_check"`
```
Comment (itzg) [inline process_mon.sh]: Since `$process_check` will contain either 0 or 1 from the `grep -c` this doesn't seem to grab the `etimes` from the intended pid.

**Comment by tcaddy** [comment]
Oops thought it was getting a PID.  Let me try again

File: ubuntu_updates_check.sh [hunk 2]
```diff
@@ -23,15 +23,15 @@
...
 
 if [ $? -ne 0 ]; then
     echo "Failed to retrieve a number of pending updates"
     exit 100
 fi
 
-PENDING_OTHER=$(echo "${OUTPUT}" | cut -d ";" -f 1)
-PENDING_SECURITY=$(echo "${OUTPUT}" | cut -d ";" -f 2)
+PENDING_OTHER=$(echo "${OUTPUT}" | grep -v "Listing..." | grep -v "/$(lsb_release -cs)-security" | wc -l)
+PENDING_SECURITY=$(echo "${OUTPUT}" | grep -v "Listing..." | grep "/$(lsb_release -cs)-security" | wc -l)
```
Comment (itzg) [inline ubuntu_updates_check.sh]: I did a quick test and noticed that some packages on my system are available via both channels, so the pattern here needs to match a leading slash or comma in order to match:

```
ruby1.9.1-dev/trusty-updates,trusty-security 1.9.3.484-2ubuntu1.13 amd64 [upgradable from: 1.9.3.484-2ubuntu1.7]
```
